README.md

```diff
@@ -1,18 +1,22 @@
 # Faraday
 
 Faraday is an HTTP client lib that provides a common interface over many
-adapters (such as Net::HTTP) and embraces the concept of Rack middleware when
+adapters (such as `Net::HTTP`, the default adapter) and embraces the concept of Rack middleware when
 processing the request/response cycle.
 
 Faraday supports these adapters:
 
-* Net::HTTP
+* [`Net::HTTP`][net_http]
+* [`Net::HTTP::Persistent`][net_http_persistent] _(experimental)_
 * [Excon][]
 * [Typhoeus][]
 * [Patron][]
 * [EventMachine][]
 * [HTTPClient][]
 
+[net_http]: http://ruby-doc.org/stdlib/libdoc/net/http/rdoc/Net/HTTP.html
```

Comment:
Could we move the URLs down to group them with the rest of URLs such as for Excon? Thanks